File: Assignment 9/Problem1.java
//Accept N number from user and return frequency of even numbers.

import java.lang.*;
import java.util.*;

class Numbers
{
	public int Frequency(int Brr[])
	{
		int iFrequency =0;
		
		for(int iCnt=0; iCnt<Brr.length;iCnt++)
		{
			if(Brr[iCnt]%2==0)
			{
				iFrequency++;
			}
		}
		
		return iFrequency;
	}
}

class Problem1
{
	
	public static void main(String a[])
	{
		Numbers nobj = new Numbers();
		Scanner sobj =new Scanner(System.in);
		
		System.out.println("Enter Number of Elements");
		int iSize = sobj.nextInt();
		
		int Arr[] = new int[iSize];
		
		System.out.println("Enter Elements of Array");
		
		for(int iCnt=0;iCnt<iSize;iCnt++)
		{
			Arr[iCnt] = sobj.nextInt();	
		}
		
		int iRet = nobj.Frequency(Arr);
		
		System.out.println("Frequency of even number is "+iRet);
		
	}
}

File: Assignment 11/Problem1.c
//Accept N number from user and return the largest number

#include<stdio.h>
#include<stdlib.h>

int Maximum(int Arr[], int iLength)
{
    int iCnt = 0;
	int iNo =Arr[0];
	
  
    for(iCnt =0; iCnt<iLength ; iCnt++)
    {	
            if(Arr[iCnt]>iNo)
			{
				iNo=Arr[iCnt];
			}
    }
	return iNo;
}

int main()
{
    int iSize = 0, iCnt = 0;
    int *ptr = NULL;   
    int iRet=0;

    printf("Enter number of elements\n");
    scanf("%d",&iSize);

    ptr = (int *)malloc(sizeof(int) * iSize);

    printf("Enter the values\n");
	
    for(iCnt = 0; iCnt < iSize; iCnt++)
    {
        scanf("%d",&ptr[iCnt]);
    }

    iRet=Maximum(ptr, iSize);
	
	printf("Largest number is %d\n",iRet);
	
    return 0;
}